### DISCUSSION TITLE
Uscita Nuova Versione

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#484] [Richieste di aiuto] [answered]
Salve, entro quando uscirà la nuova versione?

**Comment by trinko** [comment, marked as answer]
Entro la fine di questa settimana. Devo ancora fare un po' di test su alcune procedure.

**Reply by Lucaccc** [reply]
> Entro la fine di questa settimana. Devo ancora fare un po' di test su alcune procedure.

Non include nulla di nuovo oltre alla gestione notifiche e alle presenze fuori aula, giusto?

**Reply by trinko** [reply]
Oltre quello c'è il completamento dell'accesso con OTP, correzioni varie e aggiornamento dei moduli Symfony (tra cui alcuni aggiornamenti di sicurezza).
